pkg/controller/registry/resolver/rbac.go

```diff
@@ -90,6 +90,21 @@ func RBACForClusterServiceVersion(csv *v1alpha1.ClusterServiceVersion) (map[stri
 			permissions[permission.ServiceAccountName] = NewOperatorPermissions(serviceAccount)
 		}
 
+		operatorConditionRBAC := []rbacv1.PolicyRule{
+			{
+				Verbs:         []string{"get"},
+				APIGroups:     []string{"operators.coreos.com"},
+				Resources:     []string{"operatorconditions"},
+				ResourceNames: []string{csv.GetName()},
+			},
+			{
+				Verbs:         []string{"get,update"},
```

Comment:
+patch?

Reply:
I think that's fair, I'll add it in an update.

Reply:
Added in the latest version of the PR.